packages/editor-ui/src/components/CodeNodeEditor/CodeNodeEditor.vue

```diff
@@ -354,16 +354,8 @@ export default defineComponent({
 		const [languageSupport, ...otherExtensions] = this.languageExtensions;
 		extensions.push(this.languageCompartment.of(languageSupport), ...otherExtensions);
 
-		let doc = this.modelValue ?? this.placeholder;
-
-		const lines = doc.split('\n');
-
-		if (lines.length < this.rows) {
-			doc += '\n'.repeat(this.rows - lines.length);
-		}
-
 		const state = EditorState.create({
-			doc,
+			doc: this.modelValue ?? this.placeholder,
```

Comment:
is this reverting a change? why are we adding the placeholder here?

Reply:
no, we are not changing this line, this is how it was set on master

![image](https://github.com/n8n-io/n8n/assets/88898367/b3b5e886-ba9e-45b6-aff0-5878040d0f6b)

Reply:
so removing placeholder was a bug?

Reply:
do we need a test for this then?

Reply:
no, placeholder was never removed, bug was in this line `@valueChanged="valueChangedDebounced"` that remained in sql editor template,

after, on master, it was updated to `@update:modelValue="valueChangedDebounced"` previous method for adjusting editor height stop working, so I had to revisit solution, now it's more cleaner